catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/CatalogDorisIT.java

```diff
@@ -122,7 +123,10 @@ private void clearTableAndSchema() {
     NameIdentifier[] nameIdentifiers =
         catalog.asTableCatalog().listTables(Namespace.of(metalakeName, catalogName, schemaName));
     for (NameIdentifier nameIdentifier : nameIdentifiers) {
-      catalog.asTableCatalog().dropTable(nameIdentifier);
+      // The table [test_add_index_*]'s state is SCHEMA_CHANGE, cannot be dropped.
+      if (!nameIdentifier.name().contains("test_add_index")) {
+        catalog.asTableCatalog().dropTable(nameIdentifier);
+      }
     }
```

Comment:
Because you have already drop the schema cascade in the 131 line, this code block can be removed.